### PR TITLE
fix missing helper method bug; hide profile links for non social

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: app/models/presenters/content_module_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  # Wrapper for BitCore::ContentModule instances.
+  class ContentModulePresenter
+    NON_DIDACTIC_TOOLS = %w( Tools::Messages Tools::Goals )
+
+    def initialize(content_module)
+      @content_module = content_module
+    end
+
+    def not_primary_or_not_didactic?
+      tool_type = @content_module.tool && @content_module.tool.type
```
Comment (michaelwehrley) [inline app/models/presenters/content_module_presenter.rb]: What made you choose this over `tool_type = @content_module.tool.try(:type)`?

Comment (ericcf) [inline app/models/presenters/content_module_presenter.rb]: It's vanilla Ruby that doesn't depend on Active Support, and the spec doesn't need to require additional files.

File: spec/models/presenters/content_module_presenter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      context "when the content module is not primary and is didactic" do
+        it "returns true" do
+          content_module = double("content module", position: 2, tool: nil)
+          presenter = ContentModulePresenter.new(content_module)
+
+          expect(presenter.not_primary_or_not_didactic?).to eq true
+        end
+      end
+
+      context "when the content module is not primary and is not didactic" do
```
Comment (michaelwehrley) [inline spec/models/presenters/content_module_presenter_spec.rb]: Do you need this spec? If it isn't `1` then the return value is always `true`.

**Comment by michaelwehrley** [comment]
:eyes: good!

File: spec/models/presenters/content_module_presenter_spec.rb (hunk 2 above)
Comment (ericcf) [inline spec/models/presenters/content_module_presenter_spec.rb]: It's supposed to be, but that's why I'm testing :-) I'm trying to treat the method like a black box.

**Pull request merged**